colors/Photon.gpl

```diff
@@ -0,0 +1,56 @@
+GIMP Palette
+Name: Firefox/Photon
```

Comment:
Maybe we could put some [extra documentation](https://github.com/portnov/color-palette/wiki/Gimp-palette-format-extensions#palette-metainformation) here, like the url to this repo?

Reply:
Good idea!